src/nomadic/summarize/main.py

```diff
@@ -0,0 +1,805 @@
+        .query("mutation not in @remove_mutations")
+    )
+
+    # Filter out false positives
+    analysis_df = filter_false_positives(analysis_df)
+    analysis_df.to_csv(f"{output_dir}/summary.variants.analysis_set.csv", index=False)
+
+    # Then we will compute prevalence
+    prev_df = compute_variant_prevalence(analysis_df)
+    prev_df.to_csv(f"{output_dir}/summary.variants.prevalence.csv", index=False)
+
+    for col in prevalence_by:
+        prev_by_col_df = compute_variant_prevalence_per(
+            analysis_df, master_metadata, [col]
+        )
+        prev_by_col_df.to_csv(
+            f"{output_dir}/summary.variants.prevalence-{col}.csv", index=False
+        )
+
+    # --------------------------------------------------------------------------------
+    # Gene deletion analysis
+    #
+    # --------------------------------------------------------------------------------
+
+    log.info("Calculate gene deletions...")
+    gene_deletion_df = gene_deletions(coverage_df, ["hrp2", "hrp3"])
+    gene_deletion_df.to_csv(f"{output_dir}/summary.gene_deletions.csv", index=False)
+
+    prev_gen_deletions_df = gene_deletion_prevalence_by(
+        gene_deletion_df, master_metadata, []
+    )
+    prev_gen_deletions_df.to_csv(
+        f"{output_dir}/summary.gene-deletions.prevalence.csv", index=False
+    )
+
+    for col in prevalence_by:
+        prev_gen_deletion_by_col_df = gene_deletion_prevalence_by(
+            gene_deletion_df, master_metadata, [col]
+        )
+        prev_gen_deletion_by_col_df.to_csv(
+            f"{output_dir}/summary.gene-deletions.prevalence-{col}.csv", index=False
+        )
+
+    master_metadata.to_csv(f"{output_dir}/{summary_name}.metadata.csv", index=False)
+
+    # --------------------------------------------------------------------------------
+    # Dashboard
+    #
+    # --------------------------------------------------------------------------------
+
+    if show_dashboard:
+        dashboard = BasicSummaryDashboard(
+            summary_name,
+            throughput_csv=f"{output_dir}/summary.throughput.csv",
+            samples_csv=f"{output_dir}/summary.samples_qc.csv",
+            samples_amplicons_csv=f"{output_dir}/summary.samples_amplicons_qc.csv",
+            coverage_csv=f"{output_dir}/summary.experiments_qc.csv",
+            analysis_csv=f"{output_dir}/summary.variants.analysis_set.csv",
+            gene_deletions_csv=f"{output_dir}/summary.gene_deletions.csv",
+            master_csv=f"{output_dir}/{summary_name}.metadata.csv",
+            geojson_glob=f"metadata/{summary_name}-*.geojson",
+            location_coords_csv=f"metadata/{summary_name}.coords.csv",
+            settings=settings,
+        )
+        print("Done.")
+
+        print("")
+        print("Launching dashboard (press CNTRL+C to exit):")
+        print("")
+        debug = bool(os.getenv("NOMADIC_DEBUG"))
+        if not debug:
+            webbrowser.open("http://127.0.0.1:8050")
```

Comment:
I get a "Firefox can’t establish a connection to the server at 127.0.0.1:8050." with this implementation, which only corrects on refresh. 

Could we use the timer implementation in `nomadic realtime` e.g. lines 92-120 of `src/nomadic/realtime/dashboard/builders.py`?

If you want I can also make this commit, just let me know.